tests/manager/addCustomer/managerCanAddNewCustomer.spec.js

```diff
@@ -23,6 +26,30 @@ usage:
  const lastName = faker.person.LastName();
```

Comment:
There is a typo in the variable name `lastName` when using faker. It should be `faker.person.lastName()` instead of `faker.person.LastName()` to match the correct method name.